keyboards/modelh/info.json

```
@@ -0,0 +1,1103 @@
+{
+  "keyboard_name": "Model M",
+  "manufacturer": "IBM",
+  "url": "modelh.club",
+  "maintainer": "jhawthorn",
+  "usb": {
+    "vid": "0xFEED",
+    "pid": "0xB155",
+    "device_version": "1.0.0"
+  },
+  "debounce": 5,
+  "diode_direction": "COL2ROW",
+  "matrix_pins": {
+    "cols": [
+      "A10",
+      "A9",
+      "A8",
+      "B15",
+      "B14",
+      "B13",
+      "B12",
+      "B11",
+      "B10",
+      "B1",
+      "B0",
+      "A7",
+      "A6",
+      "A5",
+      "A4",
+      "A3"
+    ],
+    "rows": [
+      "B6",
+      "B5",
+      "B4",
+      "A15",
+      "B3",
+      "A0",
+      "A2",
+      "A1"
+    ]
+  },
+  "indicators": {
+    "num_lock": "B9",
+    "caps_lock": "B8",
+    "scroll_lock": "B7"
+  },
+  "processor": "STM32F103",
+  "bootloader": "stm32-dfu",
```

this is at direct odds with the config in your rules.mk.

```suggestion
  "bootloader": "stm32duino",
```